store/dataStore.ts

```diff
@@ -30,4 +65,26 @@ export const useDataStore = create<DataStore>((set) => ({
       }),
     ),
   resetUser: () => set({ user: initialState.user }),
+  updateNFLTeams: (teamName, teamLogo) =>
+    set(
+      produce((state: IDataStoreState) => {
+        state.NFLTeams.teamName = teamName;
```

Comment:
w: Find a way to set these in 1 call.

Reply:
Updated in this object:

`
set(
      produce((state: IDataStoreState) => {
        {
          state.NFLTeam.teamName = teamName;
          state.NFLTeam.teamLogo = teamLogo;
        }
      }),
    ),
    `